coriolisclient/v1/endpoint_storage.py

```diff
@@ -28,12 +28,12 @@ class EndpointStorageManager(base.BaseManager):
     def __init__(self, api):
         super(EndpointStorageManager, self).__init__(api)
 
-    def list(self, endpoint, environment=None):
+    def get(self, endpoint, environment=None):
```

Comment:
Considering we might well want to add some "real get functionality" per each storage domain, and we're technically listing results in this call, please leave it named to `list`.

To make the library as usable as possible, I recommend we just make this `endpoint_storage.list` method return only the list of storage domain names/IDs, and declare a separate `endpoint_storage.get_default` method (you don't need to make its own CLI command for the default at the moment)